src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    public Truck(String machineType) {
+        super(machineType);
+    }
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck" + " " + machineType + " " + "started its work.");

```

Comment:
The message should be more informative and in proper English. Instead of concatenating strings with spaces, you can use a formatted string. For example: `System.out.println("Truck " + machineType + " started its work.");`